tests/tests_C2_economic_functions.py

```diff
@@ -0,0 +1,86 @@
+import src.C2_economic_functions as e_functions
+
+project_life = 20
+wacc = 0.1
+investment_t0 = 220000
+tax = 0.15
+lifetime = [20, 15]
+present_value = 295000
+fuel_keys = {
+    "fuel_price": 1.3,
+    "fuel_price_change_annual": 0,
+    "project_lifetime": project_life,
+    "wacc": wacc
+}
+
+
+def test_annuity_factor_calculation():
```

Comment:
```suggestion
def test_annuity_factor():
```

Reply:
I'd suggest to name the test function in the following pattern:
`test_a_b()`
a = function name, here annuity_factor
b = add on in case you test something specific which is not the case here as `annuity_factor()` is very basic (see comment `capex_from_investment()`